hw/ip/otp_ctrl/data/BUILD

```diff
@@ -233,6 +233,8 @@ otp_json(
                     C = "0x0, 0x0, 0x0, 0x0",
                     D = "0x0, 0x0, 0x0, 0x0",
                 ),
+                # By default, ROM_EXT's recovery feature should be disabled.
+                "OWNER_SW_CFG_ROM_EXT_RECOVERY_EN": otp_hex(CONST.HARDENED_BYTE_FALSE),
```

Comment:
since we reserve 4 bytes, maybe we should use `HARDENED_FALSE` which is slightly heavier (in terms of hamming weight hehe).

Reply:
Done!